lightning/src/ln/channel.rs

```diff
@@ -6189,33 +6203,6 @@ where
 		Ok(())
 	}
 
-	#[rustfmt::skip]
-	fn funding_tx_constructed<L: Deref>(
-		&mut self, funding: &mut FundingScope, funding_outpoint: OutPoint,
-		holder_commitment_transaction_number: u64, logger: &L,
-	) -> Result<msgs::CommitmentSigned, AbortReason>
-	where
-		L::Target: Logger
-	{
-		funding.channel_transaction_parameters.funding_outpoint = Some(funding_outpoint);
-
-		if !is_splice {
-			self.assert_no_commitment_advancement(holder_commitment_transaction_number, "initial commitment_signed");
-			self.channel_state = ChannelState::FundingNegotiated(FundingNegotiatedFlags::new());
-		}
```

Comment:
This method might still be useful for DRYing up async signing. Alternatively, could keep it but move these lines to `Channel::funding_tx_constructed`.